ofctrl/fgraphFlow.go

```diff
@@ -556,6 +893,19 @@ func (self *Flow) install() error {
 
 			log.Debugf("flow install: added output port instr: %+v", instr)
 		}
+	case "empty":
+		instr := self.NextElem.GetFlowInstr()
+		if instr != nil {
+			err := self.installFlowActions(flowMod, instr)
+			if err != nil {
+				return err
+			}
+			if len(instr.(*openflow13.InstrActions).Actions) > 0 {
+				flowMod.AddInstruction(instr)
+			}
```

Comment:
installFlowActions always return nil, it's better to follow the function impl style or add some debug log when err.